packages/sui-critical-css/src/extract-from-app.js

```diff
@@ -27,23 +29,25 @@ export async function extractCSSFromApp({routes, config = {}}) {
 
     manifest[pathKey] = cssFileName
 
-    const configForMobileDevice = devices.m
-
     const css = await extractCSSFromUrl({
       url,
       ...configForMobileDevice
-    })
+    }).catch(() => '')
```

Comment:
what about logging the error before returning an empty string? I'm afraid that we might hide an actual error